spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkTableUtil.java

```diff
@@ -20,6 +20,7 @@
 
 import static org.apache.spark.sql.functions.col;
 
+import java.io.FileNotFoundException;
```

Comment:
Not a big deal since this change set is small, but usually we only do a change in one Spark version at a time, review and then backport.